server/services/user.ts

```diff
@@ -3,11 +3,18 @@ import memoize from 'memoizee'
 
 import UserModel from '../models/User'
 import { ModelId } from '../../types/interfaces'
+import { SerializerOptions } from '../utils/logger'
 
 interface GetUserOptions {
   includeToken?: boolean
 }
 
+export function serializedUserFields(): SerializerOptions {
+  return {
+    mandatory: ['_id', 'reference', 'name', 'use'],
```

Comment:
These fields don't exist on users.

Reply:
Weird I thought I'd updated this, changed